WordPress/Classes/ViewRelated/NUX/LoginFields.h

```diff
@@ -11,6 +11,8 @@
 @property (nonatomic, copy) NSString *multifactorCode;
 @property (nonatomic, assign) BOOL userIsDotCom;
 @property (nonatomic, assign) BOOL shouldDisplayMultifactor;
+@property (nonatomic, assign) NSInteger safariStoredUsernameHash;
+@property (nonatomic, assign) NSInteger safariStoredPasswordHash;
```

Comment:
Probably a good chance to add nullability annotations to `LoginFields`, or even convert to Swift as it's basically a struct (even though it'd need to be a `class` for Obj-C compatibility)